GVFS/GVFS.Service/Handlers/RequestHandler.Windows.cs

```diff
@@ -0,0 +1,37 @@
+﻿using GVFS.Common.NamedPipes;
+using GVFS.Common.Tracing;
+using GVFS.Service.Handlers;
+using System.Runtime.Serialization;
+
+namespace GVFS.Service.Handlers
+{
+    public class WindowsRequestHandler : RequestHandler
+    {
+        private const string EnablePrjFltRequest = "attach volume";

```

Comment:
I would add `Name` or `Description` to the end of each of these `...Request` fields.